src/Test/L0/ExtensionManagerL0.cs

```diff
@@ -63,6 +63,12 @@ public void LoadsTypes()
                 AssertContains<Microsoft.VisualStudio.Services.Agent.Worker.LegacyTestResults.IResultReader>(
                     manager,
                     concreteType: typeof(Microsoft.VisualStudio.Services.Agent.Worker.LegacyTestResults.JUnitResultReader));
+                AssertContains<Microsoft.VisualStudio.Services.Agent.Worker.Maintenance.IMaintenanceServiceProvider> (
+                    manager,
+                    concreteType: typeof(Microsoft.VisualStudio.Services.Agent.Worker.Build.WorkspaceMaintenanceProvider));
+                AssertContains<Microsoft.VisualStudio.Services.Agent.Worker.Maintenance.IMaintenanceServiceProvider>(
+                    manager,
+                    concreteType: typeof(Microsoft.VisualStudio.Services.Agent.Worker.Release.ReleaseDirectoryManager));
             }
```

Comment:
Are there any other of these that do not have tests?

Reply:
Yes, lots. I didn't want to clutter this PR up with those, but we have dozens of extensions now.